### PR TITLE
feat: file picker

### PR DESCRIPTION
**Pull request by EllaMartirosyan**
## Checklist
- [x] Tests
- [x] Stories
- [x] Lint
- [x] Prettier

## What I did

- [ ] BREAKING CHANGE
- [x] feature request
- [ ] bug fix
- [ ] documentation

More information:

### CODE REVIEW

File: packages/react-components/src/lib/file-picker/file-picker.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+export const DarkTheme: Story = () => {
+  return (
+    <FilePicker
+      theme={{
+        primary: 'blue',
+        background: 'black',
+        textOnBackground: 'white',
+        selectionBackground: '#455570',
+      }}
+      isDarkTheme={true}
```
Comment (alebinson) [inline packages/react-components/src/lib/file-picker/file-picker.stories.tsx]: Do not think we should expose **isDarkTheme** theme is our own

**Review by alebinson — CHANGES_REQUESTED** [review]
Only a small one

File: packages/react-components/src/lib/file-picker/file-picker.stories.tsx (hunk 1 above)
Comment (EllaMartirosyan) [reply packages/react-components/src/lib/file-picker/file-picker.stories.tsx]: Fixed

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**